prow/jobs/test-infra/check_config_files.yaml

```diff
@@ -0,0 +1,17 @@
+presubmits:
+  kyma-project/test-infra:
+  - name: prow/test-infra/check_config
+    trigger: "(?m)^/yamllint"
+    rerun_command: "/yamllint"
+    run_if_changed: "prow/*.yaml"
+    decorate: true
+    optional: true
```

Comment:
I think this shouldn't be optional. Wrong configuration can lead to problems for others as well.

Reply:
I have changed it to `optional` since it was mutually exclusive with `run_if_changed`. Do you think we should check it every time, even if the files were not modified?